components/search/SearchBar.vue

```diff
@@ -70,8 +93,42 @@ export default class SearchBar extends mixins(
     })
   }
 
+  mounted() {
+    window.addEventListener('resize', this.onResize)
+  }
+
+  beforeDestroy() {
+    window.removeEventListener('resize', this.onResize)
+  }
```

Comment:
can use ```useWindowSize``` instead

https://vueuse.org/core/useWindowSize/

Reply:
Thanks. It works.